README.md

```diff
@@ -50,55 +50,49 @@ To use the latest build (pre-release), don't forget to enable the download of sn
 
 ### Usage Example
 We provide database drivers that intercept calls to real database drivers and swap out secret IDs for actual login credentials.
-This prevents hard-coding database credentials into your application code. This can be integrated into your app through a few
-configuration file changes. Here is an example for making this work with your c3p0 config:
+This prevents hard-coding database credentials into your application code. 
 
-```properties
-# c3p0.properties
 
```

Comment:
It would be nice to have a sort sentence or two that introduces and explains the example.